src/sparseml/transformers/nm_training_arguments.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
Very general comment, is there a reason for forking the SparseML repository, creating branch locally in your fork, and then finally merging the PR into original repository's master?
I guess that for people who are inside NM (not external contributors), it would be advisable to work directly within our repo, since then other team members can e.g. quickly merge your branch or test your branch without cloning your fork. Or maybe there are some hidden benefits of your method which I am just failing to see? :)

Reply:
I was just following the guidelines for contributions, no strong reason. But noted, will work from our repo for following changes.